### ISSUE TITLE
Holotype locality missing for some HBMW treatments in GBIF

### ISSUE THREAD

**Issue by n8upham** [plazi/community#157]
I noticed for the species Holochilus sciureus Wagner, 1842, the type locality of 
>  “Rio Francisco [= Rio Sao Francisco] in Brasilien.” Regarded by P. Hershkovitz in 1955 to be in Minas Gerais State, Brazil.
given in the HBMW treatment (https://tb.plazi.org/GgServer/html/03F06D13FF0B20C108AE1F290A84FB0C) is not included in the corresponding material citation entry in GBIF (https://www.gbif.org/occurrence/3818814319) which instead says "Recorded at unknown location / unknown date".

Maybe this is an issue with parsing the type localities?  Not sure how widespread the issue might be.  Thanks!

**Comment by myrmoteras** [comment]
@n8upham this in fact might be a point of discussion: Do you happen to have a student we could train to make the material citations properly, also for other publications?

**Comment by debpaul** [comment]
Hm. I wonder if this is where expanding Global Names to work on localities (place names) would help to find these locations in publications. @dimus?

**Comment by myrmoteras** [comment]
this is a question of getting the boundaries of the material citations right, not single locations. and in this specific case, there is a also a comment, ie interpretation included.

**Comment by flsimoes** [comment]
Actually the boundaries are pretty much set, bar some exceptions here and there. The main missing part is the actual parsing of them, which takes some effort and, as Donat said, interpretation.

**Comment by n8upham** [comment]
@myrmoteras Yes this is possible for us to find a student to work on parsing the type localities more specifically.  What is the size of the corpus on which you'd need this kind of help, at least for mammals?

**Comment by flsimoes** [comment]
> @myrmoteras Yes this is possible for us to find a student to work on parsing the type localities more specifically. What is the size of the corpus on which you'd need this kind of help, at least for mammals?

While we wait for Donat's own reply, I think that would apply to all the remaining chapters from the HBMW.

**Comment by flsimoes** [comment]
@myrmoteras Cleaning up some stale issues, not sure what to do with this one
